gdx/test/com/badlogic/gdx/utils/BinaryHeapTest.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright 2024 AUTHOR
```

Comment:
See https://github.com/search?q=repo%3Alibgdx%2Flibgdx+%22see+authors+file%22&type=code for examples of what it looks like. The other classes are better but missing the capital S.

Reply:
Thank you ! I changed the licences